style.css

```diff
@@ -0,0 +1,53 @@
+* {
+	box-sizing: border-box;
+}
+body {
+    background-color: #FFFFFF;
+    margin: 0;
+    padding: 0 10% 0 10%;
+    font-family: 'Questrial', sans-serif;
+}
+h1 {
+    font-size: 3em;
+    font-weight: bold;
+}
+.underline {
+    border-bottom: 5px solid #FFFFFF;
+    padding: 4px;
+}
+.underline:hover {
+    border-bottom: 5px solid #000000;
+}
+.navbar {
+    color: #000000;
+    font-size: 2em;
+    font-weight: bold;
+    display: flex;
+    justify-content: space-evenly;
+    padding: 10px 0 20px 0;
+}
+.image {
+    display: flex;
+    justify-content: center;
+} 
+img {
+    width: 90%;
+}
+.bags {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
Using CSS Grid? Nice

Reply:
thanks! Rich helped me out, and i really like it! need to read up on it!